### PR TITLE
Tokenizer

### PR DESCRIPTION
**Pull request by furunkel**

### CODE REVIEW

**Comment by hlibbabii** [comment]
@furunkel do you know why the metrics are so bad?

**Comment by furunkel** [comment]
I passed a compiled regexp :facepalm:, but it only seems to work with regexp strings. Anyway, now it should work.

**Comment by furunkel** [comment]
@hlibbabii do you think this is ready to be merged?

File: bohr/snorkel_utils.py [hunk 1]
```diff
@@ -24,12 +25,14 @@ class Label(Enum):
...
+_tokenizer = RegexpTokenizer(r"[\s_\.,%#/\?!\-\'\"\)\(\]\[\:]", gaps=True)
 
-def safe_word_tokenize(text: Any) -> Set[str]:
+def safe_tokenize(text: Any) -> Set[str]:
     if text is None: return set()
     if pd.isna(text): return set()
 
-    return word_tokenize(str(text).lower())
+    tokens = _tokenizer.tokenize(str(text).lower())
+    return tokens
```
Comment (hlibbabii) [inline bohr/snorkel_utils.py:36]: Did you leave this `print` on purpose?

**Review by hlibbabii — COMMENTED** [review]
Looks good to me!

File: bohr/snorkel_utils.py (hunk 1 above)
Comment (furunkel) [reply bohr/snorkel_utils.py:36]: Already removed it :smile:

**Pull request merged**